### PR TITLE
initial commit - K8s custom resource type

### PR DESCRIPTION
**Pull request by srinivasa-vasu**
Signed-off-by: Srinivasa Vasu <srinivasan.surprise@gmail.com>

### CODE REVIEW

File: srinivasa-vasu-concourse-k8s.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: git
```
Comment (YoussB) [inline srinivasa-vasu-concourse-k8s.yml]: should this be `concourse-k8s`?

**Review by YoussB — CHANGES_REQUESTED** [review]

File: srinivasa-vasu-concourse-k8s.yml (hunk 1 above)
Comment (srinivasa-vasu) [reply srinivasa-vasu-concourse-k8s.yml]: done

**Comment by YoussB** [comment]
seems like there is something wrong with the commits. 2 of them are not signed. If you want I can squash and merge them. Also, the name doesn't seem to be changed.

**Comment by srinivasa-vasu** [comment]
Pls check now

**Comment by YoussB** [comment]
My Bad, I added the comment under the `name: git` but on another look, it looked like it is on the name of the file. Now as per the Readme, the name of the file should be `[OWNER_NAME]-[RESOURCE_TYPE_NAME].yml` that should be `srinivasa-vasu-k8s` and the name inside the yaml should look the same as the resource type name when used in a pipeline, thus `k8s`

Unless you disagree, I will update both of them and merge the PR. 
Please reopen if you want to change anything

**Pull request merged**